figshare/Article.py

```diff
@@ -291,9 +291,10 @@ def __get_article_metadata_by_version(self, version, article_id):
                             files = version_data['files']
 
                         version_md5 = ''
-                        version_data = standardize_api_result(version_data)
-                        version_data = sorter_api_result(version_data)
-                        json_data = json.dumps(version_data).encode("utf-8")
+                        version_data_for_hashing = metadata_to_hash(version_data)
+                        version_data_for_hashing = standardize_api_result(version_data_for_hashing)
+                        version_data_for_hashing = sorter_api_result(version_data_for_hashing)
+                        json_data = json.dumps(version_data_for_hashing).encode("utf-8")
```

Comment:
Didn't we talk about using only the value of the keys in the hashing function in case the key name changes?